drivers/volume/portworx/portworx.go

```diff
@@ -952,6 +998,68 @@ func (d *portworx) WaitDriverDownOnNode(n node.Node) error {
 	return nil
 }
 
+func (d *portworx) ValidateStoragePools() error {
+	// check if the sizes of pools match the expected sizes after storage expand
+	if len(d.autopilotRules) > 0 {
```

Comment:
I thought we had decided the query autopilotRules at runtime

https://github.com/portworx/torpedo/pull/390#discussion_r369187563

The portworx driver should not be needing to save `autopilotRules`